nixos/lib/build-vms.nix

```diff
@@ -4,15 +4,14 @@
 , # Ignored
   config ? null
 , # Nixpkgs, for qemu, lib and more
-  pkgs
+  pkgs, lib
```

Comment:
Is this change compatible with all the callers? I am concerned about `nixos-build-vms` here.

Reply:
It is, because nixos-build-vms doesn't directly call `build-vms.nix`, but `lib/testing-python.nix`.
However, I missed one call in the latter, so thanks for bringing this up.

Reply:
I can confirm `nixos-build-vms` works fine.